pkg/qos/queue_sort.go

```diff
@@ -48,13 +48,16 @@ func (*Sort) Less(pInfo1, pInfo2 *framework.PodInfo) bool {
 
 func compQOS(p1, p2 *v1.Pod) bool {
 	p1QOS, p2QOS := v1qos.GetPodQOS(p1), v1qos.GetPodQOS(p2)
-	if p1QOS == v1.PodQOSGuaranteed {
+	switch p1QOS {
```

Comment:
Hmm, I'm not sure this can improve the readability. To me, it's just stylish difference.